core/src/client/mod.rs

```diff
@@ -1005,7 +1004,7 @@ impl KakarotClient for KakarotClientImpl {
             }
         }
 
-        let kakarot_class_hash = FieldElement::from_hex_be(KAKAROT_CONTRACT_ACCOUNT_CLASS_HASH)
+        let kakarot_class_hash = FieldElement::from_hex_be(ACCOUNT_CLASS_HASH)
             .map_err(|e| {
                 KakarotClientError::OtherError(anyhow::anyhow!(
                     "Kakarot Failed to convert Kakarot custom proxy contract class hash to FieldElement: {}",
```

Comment:
Change the message to _"Kakarot Failed to convert account contract class hash to FieldElement: {}"_?

Reply:
Good idea